trino/dbapi.py

```diff
@@ -456,9 +456,8 @@ def execute(self, operation, params=None):
         else:
             self._query = trino.client.TrinoQuery(self._request, sql=operation,
                                                   legacy_primitive_types=self._legacy_primitive_types)
-            result = self._query.execute()
-        self._iterator = iter(result)
-        return result
+            self._iterator = iter(self._query.execute())
+        return self
 
     def executemany(self, operation, seq_of_params):
```

Comment:
For consistency do we also want it for `executemany`?

Reply:
added for consistency